src/test/javascript/spec/component/shared/clone-repo-button.component.spec.ts

```diff
@@ -63,14 +64,15 @@ describe('JhiCloneRepoButtonComponent', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, TranslateModule.forRoot(), NgbModule, ArtemisSharedModule, FeatureToggleModule, ClipboardModule],
-            declarations: [CloneRepoButtonComponent, MockComponent(ExerciseActionButtonComponent)],
+            imports: [ArtemisTestModule, NgbModule, FeatureToggleModule, ClipboardModule],
```

Comment:
Ngb Module could also be mocked (I think with `MockDirective(NgModel)`)

Reply:
after applying this solution it throws another errors but this workaround works well in my local. I could not find a solution for Clipboard or NgbPopover. test suite needs them. But I don't know how much of an effect that will have on it. I do not have so much experience with tests 😄 
```suggestion
            imports: [ArtemisTestModule, ClipboardModule, NgbPopoverModule],
            declarations: [CloneRepoButtonComponent,
                MockComponent(ExerciseActionButtonComponent),
                MockPipe(ArtemisTranslatePipe),
                MockPipe(SafeUrlPipe),
                MockDirective(FeatureToggleDirective),
            ]
```

Reply:
Yeah I think my suggestion was wrong, NgModel is not the same as NgbModule, but this module could still be removed so thanks for the clarification what exactly to do @imstreetspirit. We often use `MockDirective(NgbPopover)` to mock the popover, might be worth a try.